alerts/cassandra/high-error-rate.json

```diff
@@ -0,0 +1,27 @@
+{
+    "displayName": "Cassandra - High Error Rate",
+    "documentation": {
+        "content": "This alert looks at the amount of requests made and the amount of errors received and alerts if ratios of exceptions exceeds 10% of the total amount of requests. Things captured in this alert could include timeouts, write exceptions, unavailable exceptions and etc. These generally require further investigation and indicate an issue with the Cassandra node.",
```

Comment:
Slight grammar adjustment to fix the run-on sentence: 

This alert policy looks at the amount of requests made and the amount of errors received. An alert is triggered  alerts if the number of exceptions exceeds 10% of the total amount of requests.